### ISSUE TITLE
Show the same error on Windows and Mac

### ISSUE THREAD

**Issue by koraniar** [vuejs/eslint-plugin-vue#879]
**Tell us about your environment**	
* **ESLint version:**	5.14.1
* **eslint-plugin-vue version:**	5.2.2
* **Node version:**	 10.15.3

**Please show your full configuration:**

```
// https://eslint.org/docs/user-guide/configuring

module.exports = {
  root: true,
  parserOptions: {
    ecmaVersion: 2018,  
    sourceType: 'module',
    parser: 'babel-eslint'
  },
  env: {
    browser: true,
    es6: true,
  },
  extends: [
    'airbnb-base',
    'plugin:vue/recommended', 
  ],
  // required to lint *.vue files
  plugins: [
    'vue'
  ],
  // add your custom rules here
  rules: {
    // allow async-await
    'generator-star-spacing': 'off',
    // allow debugger during development
    'no-debugger': process.env.NODE_ENV === 'production' ? 'error' : 'off',
    'linebreak-style': 0
  },
  globals: {
    Atomics: 'readonly',
    SharedArrayBuffer: 'readonly',
  },
}
```

**What did you do?**	

```vue
{{ locationSelected
                  ? locationSelected.updatedOn : '' |moment('MM/DD/YYYY hh:mm a') }}
```

**What did you expect to happen?**
Show the same error or success on Mac and Windows

**What actually happened?**

On Windows it shows
118:81  warning  Multiple spaces found before '}}'  vue/no-multi-spaces
But in Mac it doesn't show error

When I put all in the same line, the warning disappears but I exceed the maximum length of 100 characters.

**Comment by ricardopolo** [comment]
Hello!  Do you have any news about this issue??

**Comment by ota-meshi** [comment]
Thank you for this issue.

I opened the PR on `vue-eslint-parser`.

https://github.com/mysticatea/vue-eslint-parser/pull/74
